app/src/components/modal/modal_your_connection/index.tsx

```diff
@@ -85,9 +85,17 @@ const ClaimLeft = styled.div`
   flex-direction: column;
   align-self: center;
 `
-const ClaimRight = styled.div`
-  margin-left: auto;
+const ClaimLeftSvg = styled.div`
   display: flex;
+  justify-content: space-between;
+  flex: 1;
+  &:hover {
+    .chevronDown {
+      path {
+        fill: ${props => props.theme.colors.primary};
```

Comment:
Hover color is wrong, please use the same hover color as arrow-back and close icon hover color